modules/channel-messenger/src/backend/index.ts

```diff
@@ -0,0 +1,22 @@
+import 'bluebird-global'
+import * as sdk from 'botpress/sdk'
+
+import { MessengerService } from './messenger'
+
+const onServerStarted = async (bp: typeof sdk) => {
+  const messengerService = new MessengerService(bp)
+  messengerService.initialize()
+}
+
+const onServerReady = (bp: typeof sdk) => {}
+
+const entryPoint: sdk.ModuleEntryPoint = {
+  onServerReady,
+  onServerStarted,
```

Comment:
Can we add onBotMount / onBotUnmount and the logic to configure messenger if it's enabled or not here? I'd like to keep it consistent with other modules, so we only check the configuration when the bot is mounted instead of everytime a request for an unknown bot is received